templates/csharp/src/Appwrite.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Do we need to add this file to .gitignore?

Reply:
yes, we need this file in order to open the solution in visual studio